tox.ini

```diff
@@ -155,7 +156,10 @@ allowlist_externals=
 commands=
     mkdir -p test-results
     # install nexus as wandb-core, requires go to be installed on the system
-    pip install ./nexus
+    # todo: this is for the CI, not to compile for the platforms we're not going to use
+    python -m build -w -n ./nexus --config-setting=--build-option=--nexus-build=linux-amd64
+    # todo: do not pin to a specific version, or bump automatically
+    pip install --force-reinstall ./nexus/dist/wandb_core-0.0.1a3-py3-none-any.whl
```

Comment:
i think this will be a bit fragile even in the short term.  maybe just a wildcard here for the version for now?

Reply:
i had a wildcard there, but then tox forces you to do /bin/bash -c ..., which you need to whitelist and also pass the env (maybe? didn't check)

Reply:
will think of a better way, i agree that it's fragile

Reply:
after a long battle with tox, introduced nox as a solution. so far loving it! (one can do what I wanted with tox, it's just so much cleaner, easier, and out-of-the-boxy with nox)

Reply:
check out how nice the logs look: https://app.circleci.com/pipelines/github/wandb/wandb/25981/workflows/069f22ee-d3c7-4399-badb-3fb0767d77b9/jobs/742972?invite=true#step-105-6301_93